components/org.wso2.carbon.identity.application.authz.xacml/src/test/java/org.wso2.carbon.identity.application.authz.xacml/handler.impl/XACMLBasedAuthorizationHandlerTest.java

```diff
@@ -0,0 +1,50 @@
+package org.wso2.carbon.identity.application.authz.xacml.handler.impl;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.application.authentication.framework.context.AuthenticationContext;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@PrepareForTest(LogFactory.class)
+public class XACMLBasedAuthorizationHandlerTest {
+
+    private XACMLBasedAuthorizationHandler xacmlBasedAuthorizationHandler;
+    HttpServletRequest mockHttpServletRequest;
+    HttpServletResponse mockHttpServletResponse;
+    AuthenticationContext context;
+
+    @BeforeClass
+    public void init() {
+        xacmlBasedAuthorizationHandler = XACMLBasedAuthorizationHandler.getInstance();
```

Comment:
Please put a single line break after starting the method

Reply:
done